account_multicurrency_revaluation_type/model/res_company.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    rate_type = fields.Selection(
```

Comment:
```suggestion
    revaluation_rate_type = fields.Selection(
```
As this field is defined on `res.company` but is only used for currency revaluation, it would make sense to add `revaluation` in its name.